docs/release_notes.md

```diff
@@ -1,5 +1,186 @@
+- Deprecate `imeta qu` (#7959).
+- Deprecate `imeta` interactive mode (#7961).
+- Deprecate server monitoring microservices (#7977).
+- Deprecate DataObjLock and DataObjUnlock (#7979).
+- Deprecate unused global variables in resource manager implementation. (#8042).
+- Deprecate `fillGenQueryInpFromStrCond()` (#8088).
+- Deprecate `forkAndExec()` (#8109).
+
+### Fixed
+
+- Avoid SIGABRT by setting pointers to null after deallocation (#3581).
+- Return errors on one-sided encryption in parallel transfer (#4984).
+- Use `ProcessType` to detect server vs pure client (#6684).
+- Rework logic in `_cllExecSqlNoResult` to return proper error codes (#7599, #7440).
+- Fix flex warning for GenQuery2 lexer rules (#7685).
+- Validate and reject invalid zone names (#7722).
+- Return non-zero exit code from `iadmin` on failure (#7734).
+- Remove `name` from exception (#7752).
+- Ignore SIGPIPE so agents do not terminate immediately (#7933, #7934).
+- Change if-statement so ticket use-count is only updated on first pass (#7967).
+- Write client and proxy user info to `ips` data file in correct order (#8001).
+- Modify JSON schema file to enforce required attributes (#8017).
+- Use `std::tolower()` with `std::transform()` correctly (#8045).
+- Fix error handling logic for heartbeat operation (#8050).
+- Return error on invalid GenQuery1 aggregate function (#8080).
+- Return error on nonexistent target for write functions in iRODS Rule Language (#8095).
+- Clean up memory in `rsDataObjChksum`, various endpoints, and iCommands (#8106).
+- Correct narrowing of floating point values (#8110).
+- Remove constructor from Cache in iRODS Rule Language to allow for `memset` (#8111).
+- Avoid segfault in `rcDisconnect` (#8120).
+- Fix libstdc++ linker error for `ienv`, `ierror`, and `ipwd` on EL8 (#8122).
+- Fix GenQuery2 column mappings for user zone (#8134).
+- Use correct GenQuery2 table aliases for permissions (#8135).
+- Fix `nullptr` to function argument and unsigned overflow (#8153).
+- Fix read/write loop in `istream` (#8166). 
+- Clear `rodsPathInp_t` instead of freeing memory (#8221).
+
+### Added
+
+- Add new keywords for creating groups via GeneralAdmin API (#2978).
+- Make checksum buffer size configurable (#7947).
+- Provide way for clients to obtain SSL/TLS certificate information (#7986).
+- Add support for Undefined Behavior Sanitizer (#8090).
+
+TODO: issue #7838 is still open, but has commits against it? 
+    - do we have other cases of this for previous releases?
+    - it's also part of the 4.3.5 milestone
+    - i see two paths forward:
+        - rename it to include something like "part 1", add it to 4.3.4 milestone, close it, and create a "part 2" issue for 4.3.5 milestone
+        - or, remove it from the release notes entirely
+        - are there other solutions to this?
+    - are release notes meant to only include "closed" issues?
+[irods/irods#7838,irods/irods#8122] CMake: remove old CentOS 7 support code
```

Comment:
@trel Thoughts on this?

Reply:
if all our commits are against an issue (which they should be), and all issues are in a milestone (which they should be), then these notes can track the issues closed.

7838 isn't in this milestone, so it doesn't seem it should be mentioned as a closed issue.

so yes, you could do a part1/part2 thing - is the work in part1 substantial enough to warrant mention in this set of release notes?   if not... then just leave out the mention.    it seems it's only CMakeLists.txt...

Reply:
7838 isn't substantial. Just can't remember if we've ever encountered this situation, which I'm betting we have.

I'm going to leave it out since it's not in this milestone.

Thinking about this a little more, the housekeeping issues are identical to this situation since the issue for those are never closed and exist in an ongoing milestone.

Reply:
Done.